packages/Spellcheck-Core.package/SpellingTextParser.class/instance/parseCamelCaseText..st

```diff
@@ -0,0 +1,19 @@
+as yet unclassified
+parseCamelCaseText: aString
+
+	| strings i startIndex endIndex |
+	
+	i := 1. startIndex := 1.	endIndex := 1.
+	strings := Dictionary new.
```

Comment:
lets find a better name for this, which explains more precisely what is stored in this variable

Reply:
Ok :)